grails-app/views/wps/show.gsp

```diff
@@ -2,14 +2,14 @@
 <html>
     <head>
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
-        <link rel="stylesheet" href="${resource(dir: 'css', file: 'imosTheme.css', absolute: true)}" type="text/css">
+        <link rel="stylesheet" href="${resource(dir: 'css', file: 'AODNTheme.css', absolute: true)}" type="text/css">
```

Comment:
We don't have to do it here and now, but given we can configure css in chef should we be looking at moving the css path to Config.groovy?